specification/communication/resource-manager/Microsoft.Communication/preview/2023-03-01-preview/Domains.json

```diff
@@ -0,0 +1,735 @@
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainProperties": {
+      "description": "A class that describes the properties of a Domains resource.",
+      "type": "object",
+      "required": [
+        "domainManagement"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of the resource.",
+          "enum": [
+            "Unknown",
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Running",
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Moving"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "Domains_ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "dataLocation": {
+          "description": "The location where the Domains resource data is stored at rest.",
+          "type": "string",
+          "readOnly": true
+        },
+        "fromSenderDomain": {
+          "description": "P2 sender domain that is displayed to the email recipients [RFC 5322].",
+          "type": "string",
+          "readOnly": true
+        },
+        "mailFromSenderDomain": {
+          "description": "P1 sender domain that is present on the email envelope [RFC 5321].",
+          "type": "string",
+          "readOnly": true
+        },
+        "domainManagement": {
+          "$ref": "#/definitions/DomainManagement"
+        },
+        "verificationStates": {
+          "description": "List of VerificationStatusRecord",
+          "type": "object",
+          "properties": {
+            "Domain": {
+              "$ref": "#/definitions/VerificationStatusRecord"
+            },
+            "SPF": {
+              "$ref": "#/definitions/VerificationStatusRecord"
+            },
+            "DKIM": {
+              "$ref": "#/definitions/VerificationStatusRecord"
+            },
+            "DKIM2": {
+              "$ref": "#/definitions/VerificationStatusRecord"
+            },
+            "DMARC": {
+              "$ref": "#/definitions/VerificationStatusRecord"
+            }
+          },
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        },
+        "verificationRecords": {
+          "description": "List of DnsRecord",
+          "type": "object",
+          "properties": {
+            "Domain": {
+              "$ref": "#/definitions/DnsRecord"
+            },
+            "SPF": {
+              "$ref": "#/definitions/DnsRecord"
+            },
+            "DKIM": {
+              "$ref": "#/definitions/DnsRecord"
+            },
+            "DKIM2": {
+              "$ref": "#/definitions/DnsRecord"
+            },
+            "DMARC": {
+              "$ref": "#/definitions/DnsRecord"
+            }
+          },
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        },
```

Comment:
If you flatten both of these, doesn't that mean they overlap/alias each other?

Reply:
Sudpito (@usvoyager ) please confirm that these propeties should not be flattened.